gh-91349: Expose the crc32 function from the lzma library (#131721)
python/cpython: Doc/library/lzma.rst

   *preset* and *filters* arguments.

.. function:: crc32(data, value=0)

   .. index::
      single: Cyclic Redundancy Check
      single: checksum; Cyclic Redundancy Check

   Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The

>    result is a positive integer. If *value* is present, it is used

Reviewer: ```suggestion
   result is a positive integer, less than :math:`2^32`. If *value* is present, it is used
```

   as the starting value of the checksum; otherwise, a default value of 0
   is used.  Passing in *value* allows computing a running checksum over the
   concatenation of several inputs.  The algorithm is not cryptographically
   strong, and should not be used for authentication or digital signatures.  Since
   the algorithm is designed for use as a checksum algorithm, it is not suitable
   for use as a general hash algorithm.

>    .. versionadded:: 3.14

Reviewer: ```suggestion
   .. versionadded:: next
```

   *preset* and *filters* arguments.

> .. function:: crc32(data[, value])

Reviewer: ```suggestion
.. function:: crc32(data, value=0)
```

   .. index::
      single: Cyclic Redundancy Check
      single: checksum; Cyclic Redundancy Check

   Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The

>    result is an unsigned 32-bit integer. If *value* is present, it is used

Reviewer: Python only has `int`, so perhaps: 
```suggestion
   result is a positive integer, in the range ``0 <= result < 2**32``.
   If *value* is present, it is used
```

This is more verbose, though, and the [`zlib.crc32`](https://docs.python.org/dev/library/zlib.html#zlib.crc32) wording just says "unsigned 32-bit integer".

   as the starting value of the checksum; otherwise, a default value of 0
   is used.  Passing in *value* allows computing a running checksum over the
   concatenation of several inputs.  The algorithm is not cryptographically
   strong, and should not be used for authentication or digital signatures.  Since
   the algorithm is designed for use as a checksum algorithm, it is not suitable
   for use as a general hash algorithm.

   .. versionadded:: 3.14

>       The result is always unsigned.

Reviewer 2: This seems strange, isn’t the versionadded directive without body enough here? (meaning the whole function is the addition)

Reviewer: ```suggestion
```
